index.js

```diff
@@ -1,251 +1,123 @@
-var kue = require('kue'),
+const fs = require("fs"),
+    kue = require("kue"),
     queue = kue.createQueue(),
-    cp = require('child_process'),
-    dbhandler = require('./dbhandler'),
-    fs = require('fs'),
-    inspect_url = require("./inspect_url"),
+    BotController = require("./lib/bot_controller"),
```

Comment:
That syntax is causing an error for me right now (Node v6.9.2) since it cannot be invoked without `new`. This works though: `botController = new (require("./lib/bot_controller"))()`.